popup.html

```diff
@@ -7,14 +7,45 @@
 </head>
 
 <body>
-    <p>This is an extension for ChatGPT search.</p>
+    
     <div id="main">
         <label for="api-key-input">Enter your ChatGPT API key</label>
         <input type="text" id="api-key-input">
         <div style="display: flex; justify-content: center;">
             <button id="save-api-key">Save</button>
         </div>
+    </div>
+    <h2>Extension Options</h2>
+
+    <div>
+        <label class="list">
+            <form>
+                <label for="model-select">Choose a model</label>
+                <select id="model-select" name="model-select">
+                </select>
+             </form>
```

Comment:
There should be a possible list of models to select.

Reply:
Not it is populated once you put you api_key directly from available model from api